### PR TITLE
Add a unit test to ensure objects within arrays are sorted.

### PR DESCRIPTION
**Pull request by MasterEric**
Before this pull request, there was no test to ensure that an object within an array had its keys sorted.

### CODE REVIEW

File: index.test.ts [hunk 1]
```diff
@@ -375,4 +375,35 @@ describe('Sort JSON', () => {
 
     expect(output).toMatchSnapshot();
   });
+
+  it('should sort JSON objects within an array', () => {
```
Comment (Gudahtt) [inline index.test.ts:379]: To differentiate this from the other test of sorting JSON objects in arrays, I'd suggest this description:

```suggestion
  it('should sort JSON objects recursively within a nested array', () => {
```

or maybe

```suggestion
  it('should sort JSON objects recursively within a non-top-level array', () => {
```

The word "recursively" was added despite that being self-evident as a hint that this pertains to the `jsonRecursiveSort` option.

Comment (Gudahtt) [inline index.test.ts:379]: Could we add a second "validate" test case as well?

So far with each fixture, I've tested the case where it _requires_ sorting, and the case where it's _already_ sorted correctly. Just to ensure that it won't ever change a valid fixture.

**Review by Gudahtt — COMMENTED** [review]
Thanks for the contribution!

There is a test that _I thought_ covered this case :thinking: The one with the description "should sort an unsorted JSON object within an array recursively".

I guess this one is different in that the array with an object isn't a top-level array. This seems like a worthwhile addition for that reason. I left a couple of suggested improvements - let me know what you think!

File: index.test.ts (hunk 1 above)
Comment (MasterEric) [reply index.test.ts]: Committed suggestion.

Comment (MasterEric) [reply index.test.ts]: Just made a commit adding this new case.

**Comment by MasterEric** [comment]
@Gudahtt Added the validation test case and renamed the existing test case.

**Review by Gudahtt — APPROVED** [review]
LGTM, thanks!

**Comment by MasterEric** [comment]
:+1:

**Pull request merged**